### PR TITLE
US6383 - Show Hide Rooms Link

### PR DESCRIPTION
**Pull request by jpc11623**
Commit of code and tests

http://mp-ci.centralus.cloudapp.azure.com/viewLog.html?buildId=16196&tab=buildResultsDiv&buildTypeId=Code_Development_SignInCheckIn_Feature

https://rally1.rallydev.com/#/27593764268d/detail/userstory/86649268228?fdp=true

### CODE REVIEW

**Comment by jpc11623** [comment]
Updated passed build:

http://mp-ci.centralus.cloudapp.azure.com/viewType.html?buildTypeId=Code_Development_SignInCheckIn_Feature&branch_Code_Development_SignInCheckIn=feature%2FUS6383-show-hide-rooms-link&tab=buildTypeStatusDiv

File: front_end/src/app/admin/events/rooms/room-list.component.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
           <div class="col-xs-11">
             <!-- TODO -->
 
-            <span class="pr-2"><a>Hide Unused Rooms</a></span> <strong>{{getOpenRooms()}}</strong> of <strong>{{getTotalRooms()}}</strong> rooms open
+            <button type="button" class="btn btn-link" (click)="toggleUnusedRooms()" >Hide Unused Rooms</button>
+            <strong>{{getOpenRooms()}}</strong> of <strong>{{getTotalRooms()}}</strong> rooms open
```
Comment (dtkocher) [inline front_end/src/app/admin/events/rooms/room-list.component.html]: Why did you use Strong tags here

File: front_end/src/app/admin/events/rooms/room-list.component.html [hunk 2]
```diff
@@ -71,7 +72,7 @@
           </tr>
         </thead>
         <tbody>
-          <tr *ngFor="let someRoom of rooms; let i = index;" class="room" (notifyDirty)="onNotifyDirty($event)" [room]="someRoom">
+          <tr *ngFor="let someRoom of rooms | ClosedRoomPipe:showClosedRooms; let i = index;" class="room" (notifyDirty)="onNotifyDirty($event)" [room]="someRoom">
```
Comment (dtkocher) [inline front_end/src/app/admin/events/rooms/room-list.component.html]: Why did you use a pipe

File: front_end/src/app/admin/events/rooms/room-list.component.ts [hunk 3]
```diff
@@ -121,4 +122,12 @@ export class RoomListComponent implements OnInit {
     this.dropdownStatus.isOpen = !this.dropdownStatus.isOpen;
   }
 
+  public toggleUnusedRooms(): void {
```
Comment (dtkocher) [inline front_end/src/app/admin/events/rooms/room-list.component.ts]: Poorly Named

**Pull request merged**